path_config.go

```diff
@@ -145,14 +146,16 @@ func (b *azureSecretBackend) pathConfigWrite(ctx context.Context, req *logical.R
 
 	if useMsGraphApi, ok := data.GetOk("use_microsoft_graph_api"); ok {
 		config.UseMsGraphAPI = useMsGraphApi.(bool)
+	} else if req.Operation == logical.CreateOperation {
+		config.UseMsGraphAPI = data.Get("use_microsoft_graph_api").(bool)
 	}
 
 	config.PasswordPolicy = data.Get("password_policy").(string)
 
-	config.RootPasswordTTL = defaultRootPasswordTTL * time.Hour
-	rootExpirationRaw, ok := data.GetOk("root_password_ttl")
-	if ok {
+	if rootExpirationRaw, ok := data.GetOk("root_password_ttl"); ok {
 		config.RootPasswordTTL = time.Second * time.Duration(rootExpirationRaw.(int))
+	} else if req.Operation == logical.CreateOperation {
+		config.RootPasswordTTL = defaultRootPasswordTTL
```

Comment:
I thought using `defaultRootPasswordTTL` was more simple than using the default value from the field schema (via `data.Get()`) by not requiring any special duration parsing. Happy to change if others don't agree.